app/helpers/application_helper.rb

```diff
@@ -30,4 +30,8 @@ def file_size_for(file, decimal_places=2)
   def format_url_safe(text)
     CGI.escape(text)
   end
+
+  def encode(string)
```

Comment:
I would try to be a bit more descriptive with the method name. Could it be merged into the method above too?